src/main/java/com/apruve/JsonUtil.java

```diff
@@ -24,17 +22,10 @@ public static JsonUtil getInstance() {
 	
 	public String toJson(Object o) {
 		try {
-			StringWriter buffer = new StringWriter();
-			HashMap<String, Object> jaxbProps = new HashMap<String, Object>();
-			jaxbProps.put("eclipselink.media-type", "application/json");
-			jaxbProps.put("eclipselink.json.include-root", false);
-			JAXBContext ctx = JAXBContext.newInstance("com.apruve.models", this
-					.getClass().getClassLoader(), jaxbProps);
-			Marshaller m = ctx.createMarshaller();
-			m.marshal(o, buffer);
-
-			return buffer.toString();
-		} catch (JAXBException e) {
+			ObjectMapper mapper = new ObjectMapper();
```

Comment:
Aren't you supposed to save and reuse your Jackson ObjectMapper? I think this is repeating a lot of work reflecting across Java objects each time.

Reply:
I don't know about "supposed to", but apparently you can.  Certainly won't hurt.